lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -636,7 +636,7 @@ def test_single_problem(self, _get_current_task):
                     unicode(self.student_2.id),
                     self.student_2.email,
                     self.student_2.username,
-                    '0.0', 'N/A', 'N/A'
+                    '0.0', '0.0', '2'
```

Comment:
*Note to reviewers:* this change is a side-effect of removing this logic: https://github.com/edx/edx-platform/pull/13102/files#diff-8c7861d1cc921711314f234296e6105cL220

Essentially, I removed all the complex `should_grade_section` code for simplicity reasons - since it's no longer needed for performance reasons.  https://github.com/edx/edx-platform/pull/13102/files#diff-8c7861d1cc921711314f234296e6105cL182

Previously, grade reports serialized `N/A` values for all problem scores in a subsection that was never attempted by a learner.  However, as soon as a learner attempts 1 problem in the subsection, all unattempted problems are then shown as `0`.  This seems confusing behavior.  In the long-run, we can/should more explicitly distinguish between unattempted problems and scores of 0.

I have spoken to @sstack22 regarding this and she is verifying with partner support on this change - just in case anyone was relying on this unexpected behavior.